README.md

```diff
@@ -1,50 +1,102 @@
 # merchandise-in-baggage
 
-**Who uses the repo/service**
+## Who uses the repo/service
 
-This is the backend service supporting two frontends (public facing and internally for administration through stride).
+merchandise-in-baggage backend service supports two frontends (public facing and internally for administration through
+stride).
 
-Both frontend services are maintained from the [merchandise-in-baggage-frontend](https://github.com/hmrc/merchandise-in-baggage-frontend) repository
-and are deployed as separate instances `merchandise-in-baggage-frontend` & `merchandise-in-baggage-admin-frontend`.
+Both `merchandise-in-baggage-frontend` & `merchandise-in-baggage-admin-frontend`
+services are maintained from
+the [merchandise-in-baggage-frontend](https://github.com/hmrc/merchandise-in-baggage-frontend) repository
+and are deployed as separate instances.
```

Comment:
Why is this here? This should be only backend related and should be moved to the frontend repo instead. Please remove.